Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Version.java

```diff
@@ -9,18 +9,23 @@
 
     private String version;
 
+    private boolean isComparable;
+
     public Version(String version) {
         if(version == null)
             throw new IllegalArgumentException("Version can not be null");
         if(!version.matches("[0-9A-Za-z]+(\\.[0-9A-Za-z]+)*"))
```

Comment:
We should not be requiring a specific version format here, the CQL spec doesn't pu9t any limitations on what a valid version string is, so just remove this check and exception

Reply:
Updated this!